README.md

```diff
@@ -50,6 +53,26 @@ This project consist of different ruby enumerable methods that emulate Ruby's in
 
 - run npm install to install the dependencies for the project.
 
+## 🔨 Testing Game Logic
```

Comment:
:exclamation: Please change this title to just **Testing**.
There is no game in this project.